packages/orbit-components/src/tmp-tailwind/Select/types.d.ts

```diff
@@ -0,0 +1,39 @@
+// Type definitions for @kiwicom/orbit-components
+// Project: http://github.com/kiwicom/orbit
```

Comment:
this file can be removed and types can be imported directly from original Stack :)